docs/monorepo.md

```diff
@@ -1,77 +1,322 @@
+
+## Overview
 
 The [main GitHub repository for Catalyst](https://github.com/bigcommerce/catalyst) is a [monorepo](https://en.wikipedia.org/wiki/Monorepo) with multiple codebases, including but not limited to the following:
 
-* The [core Next.js reference storefront application, in /core](https://github.com/bigcommerce/catalyst/tree/main/core)
-* [The component library, in /core/components/ui](https://github.com/bigcommerce/catalyst/tree/main/core/components/ui)
-* [The API client, in /packages/client](https://github.com/bigcommerce/catalyst/tree/main/packages/client)
-* [Product documentation, in /docs](https://github.com/bigcommerce/catalyst/tree/main/docs)
+- [The core Next.js reference storefront application, in `/core`](https://github.com/bigcommerce/catalyst/tree/main/core)
+- [Packages published to NPM, in `/packages`](https://github.com/bigcommerce/catalyst/tree/main/packages)
+- [Product documentation, in `/docs`](https://github.com/bigcommerce/catalyst/tree/main/docs)
 
-### The monorepo and the CLI
+It's important to note that we use several tools throughout the Catalyst monorepo, such as [Turborepo](https://turbo.build/repo/docs) and [pnpm Workspaces](https://pnpm.io/workspaces); you'll notice references to these tools as you spend more time in the repository, but most developers likely won't need to change any files, code, or configuration related to these tools in their normal Catalyst workflows.
 
-The [Catalyst CLI](https://www.npmjs.com/package/@bigcommerce/create-catalyst) installs just the Next.js application from `core/` onto your computer when you create a new Catalyst storefront.
+## Prerequisites
 
-If you are interested solely in building a headless storefront based on Catalyst, most of this monorepo is not relevant to you.
+1. **Node.js version `20.x`:** Catalyst currently supports Node.js version `20.x` ([click here to download Node.js](https://nodejs.org/en/download/package-manager))
+2. **Corepack-enabled `pnpm`:** Node.js version `20.x` ships with Corepack by default; you should be able to run `corepack enable` from your terminal to meet this prerequisite (more information on the [official Node.js Corepack documentation](https://nodejs.org/docs/latest-v20.x/api/corepack.html))
+3. **Git version `2.35.x` or later:** ([Click here to download Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git))
+4. **A BigCommerce store or sandbox:** You can [create a trial store](https://bigcommerce.com/start-your-trial/) or [spin up a developer sandbox](https://start.bigcommerce.com/developer-sandbox/) to meet this prerequisite. If you already have a BigCommerce store, that will work fine too.
+5. **Permission to create API accounts on your store:** Our [documentation on High-Risk Permissions](https://support.bigcommerce.com/s/article/User-Permissions#highrisk) can help determine whether or not you have the correct permissions for the store you want to connect to.
```

Comment:
This feels clunky to me somehow, could we maybe put a little more detail here? Like maybe add something like"Ensure the store owner has granted you the `Create store-level API Accounts` permission." prior to the link. Something about the wording here makes me feel like the link will contain info on exactly which permissions you need, when that's not really the case.